applications/tari_base_node/src/commands/cli.rs

```diff
@@ -188,5 +198,7 @@ pub fn print_banner(commands: Vec<String>, chunk_size: i32) {
     }
     println!("{}", box_line(target_line_length, false));
 
-    resize_terminal_to_fit_the_box(target_line_length, height_to_resize);
+    if resize_terminal {
```

Comment:
Since you are already checking this if, no need to pass in `resize_terminal` (See previous comment)

Reply:
Yip, remnant of last minute change...